src/App.js

```diff
@@ -1,23 +1,28 @@
 import logo from './logo.svg';
 import './App.css';
-
+import Layout from './Components/Project/Layout';
+import Navbar from './Components/Navbar/Navbar';
+import Heading from './Components/Heading';
+import Footer from './Footer/Footer';
 function App() {
```

Comment:
Make a habbit of using arrow functions:
const App = () => {
//...Your code here
}